src/Diagnostics/FxCop/System.Runtime.Analyzers/Test/Globalization/CA1309FixerTests.cs

```diff
@@ -1,14 +1,13 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
```

Comment:
It would be nice to rename the unit test files to match the name of the analyzer / fixer. Looks like analyzer and fixer are called 'UseOrdinalStringComparison*' while test files use 'CA1309*'